### PR TITLE
Blog post about HPC Carpentry at ISC19

### PR DESCRIPTION
**Pull request by ocaisa**

### CODE REVIEW

**Comment by ocaisa** [comment]
@psteinb Done at last ;)

**Comment by kariljordan** [comment]
@ocaisa thank you so much for this blog post. I tried submitting a PR from your post to remove a duplicate word.

This sentence:

> My personal view on this would be probably be "it depends".

should be

> My personal view on this would probably be "it depends".

Can you make that change?

cc @tracykteal

**Comment by ocaisa** [comment]
Thanks for catching that!

File: _posts/2019/08/2019-08-08-hpc-carpentry-at-ISC2019.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+main conference, and HETET19 had a half day slot.
+
+Probably what was most interesting for me was that I wasn't the first to discuss HPC Carpentry that day, or the last. Chris Bording
+(currently at IBM), one of the discussion panelists, talked quite a bit about HPC Carpentry. I felt he gave great context as to why
+something like HPC Carpentry is necessary, talking about the realities of the training needs of the scientists that he met in his
+various user support roles.
+
+There was supposed to be another panellist, Mike Croucher who unfortunately couldn't make it that day in the end. I had come across a
+presentation of his recently that he gave this year: "Why Nobody Cares About HPC" (unfortunately the presentation is no longer
+available online, if I had to summarise it from memory I would say: HPC UI sucks and hasn't changed in 20 years, reproducibility is
```
Comment (kariljordan) [inline _posts/2019/08/2019-08-08-hpc-carpentry-at-ISC2019.md]: @ocaisa May I make one recommendation for the section where you summarise the presentation? We found a slide deck that might be relevant, or at least related from Mike Croucher: https://mikecroucher.github.io/HPC_for_everyone/

Would this serve as a link that's useful, rather you having to recall from memory? You could summarise from this slide deck so he's accurately represented. A summary could be: "More people need HPC than ever, but it's so hard to use, reproducibility is hard, and we need to support research software engineers to help researchers navigate this space." Those sentiments are directly from the slide deck, and the slide deck is in a GitHub repo, so it's public. What do you think?

(FYI Serah is on leave and I sit on the Communications Team for The Carpentries so I'm helping review blog posts before they're added to the blog. This post is fantastic! Hopefully my recommendation will strengthen the points you've made.)

P.S. I see this is your first pull request to The Carpentries blog. Congrats!!!!!!!

Comment (ocaisa) [reply _posts/2019/08/2019-08-08-hpc-carpentry-at-ISC2019.md]: The slide deck is the same from what I can tell. I'm on vacation without a laptop but I can try do this tomorrow.

Comment (kariljordan) [reply _posts/2019/08/2019-08-08-hpc-carpentry-at-ISC2019.md]: Thanks so much for responding while you're on vacation! There's no rush. Whenever you make the changes I'll merge the post. Cheers!

**Comment by ocaisa** [comment]
@kariljordan Editing on my phone is painful...hopefully didn't introduce problems!

**Comment by kariljordan** [comment]
Thanks so much!

**Pull request merged**